examples/trap-error/trap_error.bal

```diff
@@ -0,0 +1,23 @@
+import ballerina/io;
+
+public function main() {
+    int|error result;
+    io:println("Start dividing numbers");
+
+    // An error is panicked (or thrown) when the `divideNumbers` function is executed.
+    // `trap` will catch the error which occurred in the `divideNumbers` function and assgins the `result` variable.
```

Comment:
Change to "`trap` catches the error that..."

Normally we use the word "that" in this context. "Which" is used only if you are adding additional information to the sentence and normally there is a comma before the word "which".

Also avoid future tense wherever possible. Normally if you see the word "will" in the sentence it is indicative of future tense.